app/portainer/views/endpoints/edit/endpointController.js

```diff
@@ -269,6 +271,11 @@ function EndpointController(
           $scope.state.availableEdgeAgentCheckinOptions[0].key += ` (${settings.EdgeAgentCheckinInterval} seconds)`;
         }
 
+        if (settings.FeatureFlagSettings && settings.FeatureFlagSettings['open-amt']) {
+          endpoint.ManagementInfoRaw = amtinfo.Text;
+          endpoint.ManagementInfo = JSON.parse(amtinfo.Text);
```

Comment:
Maybe it won't be neccesary if intel starts sending the response as a JSON, but for me this fails because amtinfo.Text is "Unable to launch application. Please ensure that Intel ME is present, the MEI driver is installed and that this application is run with administrator or root privileges.".

```suggestion
          try {
          	endpoint.ManagementInfo = JSON.parse(amtinfo.Text);
          } catch (e) {
          	console.error('Failed parsing AMT Info response', e);
          }
```

Reply:
yup, https://github.com/open-amt-cloud-toolkit/rpc-go/pull/23

but yes, I hit this too, and was working on a change where if there's a parse error (cos nothing's ever perfect) it can show the raw - identical code :)